lib/sisimai/bite/email/verizon.rb

```diff
@@ -0,0 +1,265 @@
+          match = 0 if mhead['from'] =~ Re0[:'vzwpix.com'][:from]
+          break
+        end
+        return nil if match < 0
+
+        require 'sisimai/mime'
+        require 'sisimai/address'
+        dscontents = [Sisimai::Bite.DELIVERYSTATUS]
+        hasdivided = mbody.split("\n")
+        rfc822list = []     # (Array) Each line in message/rfc822 part string
+        blanklines = 0      # (Integer) The number of blank lines
+        readcursor = 0      # (Integer) Points the current cursor position
+        recipients = 0      # (Integer) The number of 'Final-Recipient' header
+        senderaddr = ''     # (String) Sender address in the message body
+        subjecttxt = ''     # (String) Subject of the original message
+
+        re1        = {}     # (Ref->Hash) Delimiter patterns
+        reFailure  = {}     # (Ref->Hash) Error message patterns
+        boundary00 = ''     # (String) Boundary string
+        v = nil
+
+        if match == 1
+          # vtext.com
+          re1 = {
+            :begin  => %r/\AError:[ \t]/,
+            :rfc822 => %r/\A__BOUNDARY_STRING_HERE__\z/,
+            :endof  => %r/\A__END_OF_EMAIL_MESSAGE__\z/,
+          }
+          reFailure = {
+            # The attempted recipient address does not exist.
+            userunknown: %r/550[ ][-][ ]Requested[ ]action[ ]not[ ]taken:[ ]no[ ]such[ ]user[ ]here/x,
+          }
+          boundary00 = Sisimai::MIME.boundary(mhead['content-type']) || ''
+
+          if boundary00.size > 0
+            # Convert to regular expression
+            re1['rfc822'] = Regexp.new('\A' + Regexp.escape('--' + boundary00 + '--') + '\z')
+          end
+
+          hasdivided.each do |e|
+            if readcursor.zero?
+              # Beginning of the bounce message or delivery status part
+              if e =~ re1[:begin]
+                readcursor |= Indicators[:deliverystatus]
+                next
+              end
+            end
+
+            if readcursor & Indicators[:'message-rfc822'] == 0
+              # Beginning of the original message part
+              if e =~ re1[:rfc822]
+                readcursor |= Indicators[:'message-rfc822']
+                next
+              end
+            end
+
+            if readcursor & Indicators[:'message-rfc822'] > 0
+              # After "message/rfc822"
+              if e.empty?
+                blanklines += 1
+                break if blanklines > 1
+                next
+              end
+              rfc822list << e
+
+            else
+              # Before "message/rfc822"
+              next if readcursor & Indicators[:deliverystatus] == 0
+              next if e.empty?
+
+              # Message details:
+              #   Subject: Test message
+              #   Sent date: Wed Jun 12 02:21:53 GMT 2013
+              #   MAIL FROM: *******@hg.example.com
+              #   RCPT TO: *****@vtext.com
+              v = dscontents[-1]
+              if cv = e.match(/\A[ \t]+RCPT TO: (.*)\z/)
```

Comment:
__[rubocop]__
* Style/NumericPredicate: Use `(readcursor & Indicators[:deliverystatus]).zero?` instead of `readcursor & Indicators[:deliverystatus] == 0`. [:ref](https://github.com/bbatsov/ruby-style-guide#predicate-methods)

<!-- Comment by [SideCI](https://sideci.com) -->